src/views/WellnessCheck/components/HealthStatus/index.js

```diff
@@ -60,11 +61,11 @@ const HealthStatus = ({ currentStatus, setCurrentStatus, username, image }) => {
   return (
     <Grid container justify="center" spacing={2}>
       <Grid item xs={12} md={8}>
-        <Card className="wellness-check">
+        <Card className={styles['wellness-check']}>
```

Comment:
This is not really the most idiomatic way too use css modules. The string index access is a clunky syntax to use when regular object property access works just great. The only thing is that class names that have `-` will need to be converted to use `_`:
```suggestion
        <Card className={styles.wellness_check}>
```
In addition to being more natural/idiomatic javascript, this is also less of a stringly-typed syntax, so mistakes may be easier to catch.